tests/actions.go

```diff
@@ -328,6 +336,7 @@ func (oi *OperatorConfig) OperatorHelmSetString(m map[string]string) string {
 		"scheduler.logLevel":               "2",
 		"controllerManager.replicas":       "2",
 		"scheduler.replicas":               "2",
+		"imagePullPolicy":                  "Always",
```

Comment:
Typically, we will use the `latest` tag of operator image. `Always` can avoid using stale image